presto-main/src/test/java/com/facebook/presto/execution/TaskTestUtils.java

```diff
@@ -93,7 +94,7 @@ private TaskTestUtils()
             new PlanFragmentId(0),
             new TableScanNode(
                     TABLE_SCAN_NODE_ID,
-                    new TableHandle(CONNECTOR_ID, new TestingTableHandle()),
+                    new TableHandle(CONNECTOR_ID, new TestingTableHandle(), TestingTransactionHandle.create(), Optional.of(TestingHandle.INSTANCE)),
```

Comment:
nit: I think we should use the static `TRANSACTION_HANDLE` -- otherwise the transaction handle will be inconsistent between the `SPLIT` and the TableScanNode created in this test.